### ISSUE TITLE
更新1.4版本，安装的app卸载后打开失败。

### ISSUE THREAD

**Issue by ShAlIow** [opa334/TrollStore#356]
### No Duplicates Issue

- [X] I'm sure I've searched on the issue tracker before creating one.

### Expected Behavior?

今天我将ts更新到了1.4版本，这个版本增加了桌面卸载app的功能。于是我用ts安装一个应用，该应用可以正常打开使用。在桌面上将其卸载。再重新用ts安装此应用，应用应该仍然可以打开正常使用。（在1.4以前的版本都可以实现的一个功能）

### Current Behavior?

实际情况是用ts安装该应用，可以正常打开。桌面卸载该应用或ts中卸载该应用。再重新用ts安装此应用，在ts列表中可以看到该app，但是点击打开会提示打开失败，桌面上不会出现app图标。并且在设置中刷新图标缓存和重启桌面、重新启动手机后重复上述操作，问题仍然存在。

### Steps to Reproduce

-

### TrollStore Version

trollstore1.4

### iOS/iPadOS version

ios14.6

### iDevice Model

iphone11

### Other info of your environment

未越狱

**Comment by AnthonyGress** [comment]
Translation:
"The actual situation is that the application is installed with ts and can be opened normally. Uninstall the application on the desktop or uninstall the application in ts. Then re-install the app with ts, you can see the app in the ts list, but if you click to open, it will prompt that the opening fails, and the app icon will not appear on the desktop. And after refreshing the icon cache in the settings, restarting the desktop, and restarting the phone, repeat the above operations, the problem still exists."

I am having a similar issue. I not able to get them to even reinstall and show in the list. TS gave me an error about an existing app and the option to cancel or force install. Neither worked. Uninstalling TS did not solve the issue either.

**Comment by ShAlIow** [comment]
I filed an issue on github and there are many people in the same situation as me. I saw that 1.4 you explained has changed the way to install apps, but on my iphone11 ios14.6 device, I can only install apps. If I uninstall and reinstall the app, an error will occur. And after downgrading to 1.35, the app will still fail to install. I guess because "The FuguInstall is using stock APIs to do a placeholder installation, then fixing up their permissions and adding them to icon cache. Previously instead of the placeholder installation, TrollStore was just creating a new app bundle and copying stuff into it. "Cause of this new installation method related to placeholders that you mentioned. There is already a record in the system of uninstalling the placeholder app, and reinstallation of the same placeholder app is not allowed. When I change the bundle id, the app can be successfully installed and opened successfully. I sincerely hope that the author can give a solution to this problem in the future. This is very, very important for those of us who rely heavily on trollstore and have problems updating to version 1.4!
